### PR TITLE
EUI-8768 Register other org - Cancel link functionality

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-8768

### Change description ###
Register other org - (Functionality) Cancel link - extension of the Master Page 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/register-org/containers/register/register-org.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    fixture.destroy();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+    expect(mockRegisterOrgService.getRegistrationData).toHaveBeenCalled();
+    expect(component.registrationData).toEqual(registrationData);
+  });
+
+  it('should invoke persist registration data when component is unloaded', () => {
```
Comment (Daniel-Lam) [inline src/register-org/containers/register/register-org.component.spec.ts:75]: @johnbenjamin-hmcts - there is an additional unit test you can add: it should not invoke `persistRegistrationData` if the component is unloaded but the registration journey has been cancelled

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@johnbenjamin-hmcts - all looks good; just one suggestion for an additional unit test

File: src/register-org/containers/register/register-org.component.spec.ts (hunk 1 above)
Comment (johnbenjamin-hmcts) [reply src/register-org/containers/register/register-org.component.spec.ts:75]: @Daniel-Lam I have added the test now as you suggested.

**Review by Daniel-Lam — APPROVED** [review]

**Review by DavidJayakumar — APPROVED** [review]
LGTM

**Review by Daniel-Lam — APPROVED** [review]
@johnbenjamin-hmcts - reviewed additional changes; approved

**Review by DavidJayakumar — APPROVED** [review]

**Pull request merged**